starlette/staticfiles.py

```diff
@@ -40,7 +40,7 @@ def __init__(
         self,
         *,
         directory: PathLike = None,
-        packages: typing.List[str] = None,
+        packages: typing.List[typing.Union[str, typing.Tuple[str, str]]] = None,
```

Comment:
I tried using `@typing.overload` for here, but that doesn't seem to work on the initialiser yet.

Reply:
We might be mixing 1-string and 2-tuple items in the same list, so I think this type definition is the most appropriate. :+1: